src/store/getters.js

```diff
@@ -97,6 +97,15 @@ export default {
       : '';
   },
 
+  gameProgress: ({ game, progresses }) => {
```

Comment:
this will only return game progress when a game is active, that's what `game` is, the data for the active game (game viewed in modal)